src/styles/reset.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: $day-font;
```

Comment:
The variable `$day-font` is used here, but it is not defined in this file. Ensure that it is defined in an imported file or within this file.